imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -4525,14 +4525,46 @@ func Test_pathSymlink_errors(t *testing.T) {
 	ok = mem.Write(link, []byte(linkName))
 	require.True(t, ok)
 
-	t.Run("success", func(t *testing.T) {
-		requireErrnoResult(t, wasip1.ErrnoSuccess, mod, wasip1.PathSymlinkName,
-			uint64(file), uint64(len(fileName)), uint64(fd), uint64(link), uint64(len(linkName)))
-		require.Contains(t, log.String(), wasip1.ErrnoName(wasip1.ErrnoSuccess))
-		st, err := os.Lstat(joinPath(dirPath, linkName))
-		require.NoError(t, err)
-		require.Equal(t, st.Mode()&os.ModeSymlink, os.ModeSymlink)
-	})
+	success_cases := []struct {
```

Comment:
In Go, we use CamelCase in general.

```suggestion
	successCases := []struct {
```